xclim/indices/_agro.py

```diff
@@ -437,11 +449,12 @@ def cool_night_index(
     Parameters
     ----------
     tasmin : xarray.DataArray
-      Minimum daily temperature.
-    lat: xarray.DataArray, optional
-      Latitude coordinate.
+        Minimum daily temperature.
+    lat : xarray.DataArray or {"north", "south"}, optional
+        Latitude coordinate.
```

Comment:
```suggestion
        Latitude coordinate as an array, float or string.
```
I know this looks unnecessary, but the Indicator's docstring doesn't preserve the full annotation and only the choices are shown on RTD. With this extra text, I think it's not clearer what can be passed. See : https://xclim--1180.org.readthedocs.build/en/1180/xclim.indicators.atmos.html#xclim.indicators.atmos._temperature.cool_night_index